### PR TITLE
new question added and new style and assets added

### PR DESCRIPTION
**Pull request by vinodpatidar123**
### Please check if this PR fulfills these
- [x] The commit message is descriptive of changes
- [x] You have tried at least one local test (this includes viewing the changes on your local fork)
- [x] Docs have been added / updated as needed (for bug fixes / features) 

### What kind of change does this PR introduce? 

- [x] Typo / Doc Update
- [x] Bug Fix 
- [ ] Feature / New Item

### What is the new behavior (if this is a feature change)?

- [ ] Added Interview Technical Question
- [x] Added Interview Whiteboard Question
- [ ] Corrected existing items
- [ ] Other

### Adding new question ? Please check if this PR fulfills these

 - [x] `<title>` tag has been updated
 - [x] `<h1>` tag has been updated
 - [x] new file is `kabob-case-like-this.html`
 - [x] link to question has been added in `technical.html` or `whiteboard.html`

### CODE REVIEW

File: views/questions/whiteboard/breath-first-search.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Breath First Search | Software Interview Whiteboard Questions </title>
```
Comment (LukeSchlangen) [inline views/questions/whiteboard/breath-first-search.html]: Small typo: `Breadth` First

**Comment by vinodpatidar123** [comment]
Thanks, for your feedback. I will make the changes as soon as possible.

**Comment by LukeSchlangen** [comment]
Great work! Thank you so much for your contribution!

**Comment by vinodpatidar123** [comment]
Great to hear from you .

**Pull request merged**